odl/operator/operator.py

```diff
@@ -199,44 +406,110 @@ class Operator(with_metaclass(_OperatorMeta, object)):
 
     **Parameters:**
 
-    x : `Operator.domain` element
+    x : `Operator.domain` element-like
         An object in the operator domain to which the operator is
-        applied.
+        applied
 
-    out : `Operator.range` element
+    **Returns:**
+
+    out : `Operator.range` element-like
+        An object in the operator range holding the result of the
+        operator evaluation
+
+
+    **Dual-use evaluation:** ``_call(self, x, out=None[, **kwargs])``
+
+    Evaluate in place if ``out`` is given, otherwise out of place.
+
+    **Parameters:**
+
+    x : `Operator.domain` element-like
+        An object in the operator domain to which the operator is
+        applied
```

Comment:
I need to revise this. Actually the input is always a domain element, since that's what `_call` gets from `__call__`.